### PR TITLE
Wormhole everything

### PR DESCRIPTION
**Pull request by cedric-h**

### CODE REVIEW

File: .build.yml [hunk 1]
```diff
@@ -10,11 +9,7 @@ tasks:
...
-- nightly: |
-    rustup default nightly
+    cargo install cargo-make
+- stable: |
     cd backend/
-    cargo build
-    sudo systemctl start mongodb
-    MONGO_URL=mongodb://localhost:27017 cargo test
-    cargo fmt -- --check
+    cargo make test
```
Comment (Muirrum) [inline .build.yml:15]: I'd like CI to still run `cargo fmt -- --check`

Comment (Muirrum) [reply .build.yml:15]: ah, saw Makefile.toml. disregard.

File: src/hackstead/mod.rs [hunk 2]
```diff
@@ -1,59 +1,93 @@
...
 
-    trace!("got hackstead: {:#?}", stead);
+    if let Some(s) = hs.profile.slack_id.as_ref() {
+        fs::hard_link(&stead_path, &slack_path(s))?;
+    }
+
+    Ok(())
+}
+
+#[post("/hackstead/spy")]
```
Comment (Muirrum) [inline src/hackstead/mod.rs:45]: charming API endpoints

**Review by Muirrum — APPROVED** [review]

File: src/hackstead/mod.rs (hunk 2 above)
Comment (cedric-h) [reply src/hackstead/mod.rs:45]: read the crate-level documentation :D

**Pull request merged**